test/safe/basic-test.js

```diff
@@ -33,5 +33,11 @@ module.exports = {
       )
       done(null)
     })
+  },
+  passArgsToScript: function (done) {
+    runScripty('args:echoer', {userArgs: ['--test', 'arg passed by user']}, function (er, code, stdio) {
+      assert.includes(stdio.stdout, '--test arg passed by user')
+    })
+    done(null)
```

Comment:
This done(null) needs to be inside runScripty's callback after the assertion (or it will exit prematurely) and called with done(er) so it'll fail if anything errors